app/routers.py

```diff
@@ -167,6 +168,11 @@ async def websocket_chat_endpoint(websocket: WebSocket,
     if settings.embeddingType:
         if settings.embeddingType == schema.EmbeddingType.OPENAI:
             vectordb_args['embedding'] = OpenAIEmbedding()
+            
+        else:
+            vectordb_args['embedding'] = SentenceTransformerEmbedding() # FIXME: should have a default instantiation using schema.EmbeddingType.DEFAULT ?
```

Comment:
(Responding to the comment...)
Or may be set the value `schema.EmbeddingType.HUGGINGFACE` as default for the input objects, ChatPipelineSelector and DataPipelineSelector . That way it would get documented in the API docs aswell.

Reply:
Okay. I was wondering about just setting something like `schema.EmbeddingType.DEFAULT`, and having that point to `schema.EmbeddingType.HUGGINGFACE` in the schema. Thoughts about that? 

I figured we could always open an issue and worry about setting universal defaults like that after Wednesday.